homeassistant/components/webostv/device_trigger.py

```diff
@@ -58,7 +61,7 @@ async def async_validate_trigger_config(
 
 async def async_get_triggers(
     _hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
I agree that for now we should revert it until we finish the discussion about it

Reply:
PR to revert: https://github.com/home-assistant/core/pull/72302